### PR TITLE
Bots Upgrade

### PR DESCRIPTION
**Pull request by KarolDenst**
Bots become  hostile when shot and stay further from player

### CODE REVIEW

File: Hyper/GameEntities/AstronautBot.cs [hunk 1]
```diff
@@ -146,6 +148,48 @@ private void UpdateBotGoalsWhenHostile(Simulation simulation, float time, Scene
             movementDirection);
     }
 
+    public override void ContactCallback(ContactInfo collisionInfo, SimulationMembers simulationMembers)
```
Comment (AleksyBalazinski) [inline Hyper/GameEntities/AstronautBot.cs:151]: why is this overridden with the same exact thing?

File: Common/Settings.cs [hunk 2]
```diff
@@ -27,7 +27,7 @@ public class Settings
     /// <summary>
     /// Time elapsed since the last day start in seconds
     /// </summary>
-    public float TimeInSeconds { get; set; } = 0;
+    public float TimeInSeconds { get; set; } = 60;
```
Comment (AleksyBalazinski) [inline Common/Settings.cs:30]: we could start at noon, i,e, 60 * 5

File: Hyper/GameEntities/AstronautBot.cs (hunk 1 above)
Comment (KarolDenst) [reply Hyper/GameEntities/AstronautBot.cs:151]: There is 1 extra line that sets the bot to hostile when hit with a projectile

Comment (KarolDenst) [reply Hyper/GameEntities/AstronautBot.cs:151]: It's line 173

Comment (AleksyBalazinski) [reply Hyper/GameEntities/AstronautBot.cs:151]: Ok

**Review by AleksyBalazinski — APPROVED** [review]

**Pull request merged**